spec/models/manageiq/providers/kubernetes/container_manager/refresh_parser_spec.rb

```diff
@@ -3,6 +3,49 @@
 describe ManageIQ::Providers::Kubernetes::ContainerManager::RefreshParser do
   let(:parser)  { described_class.new }
 
+  describe "parse_namespace" do
+    it "handles simple data" do
+      expect(parser.send(:parse_namespace,
+                         RecursiveOpenStruct.new(
+                           :metadata => {
+                             :name              => "proj2",
+                             :selfLink          => "/api/v1/namespaces/proj2",
+                             :uid               => "554c1eaa-f4f6-11e5-b943-525400c7c086",
+                             :resourceVersion   => "150569",
+                             :creationTimestamp => "2016-03-28T15:04:13Z",
+                             :labels            => {:department => "Warp-drive"},
+                             :annotations       => {:"openshift.io/description"  => "",
+                                                    :"openshift.io/display-name" => "Project 2"}
+                           },
+                           :spec     => {:finalizers => ["openshift.io/origin", "kubernetes"]},
+                           :status   => {:phase => "Active"}
+                         )
+                        )).to eq(:ems_ref          => "554c1eaa-f4f6-11e5-b943-525400c7c086",
+                                 :name             => "proj2",
+                                 :ems_created_on   => "2016-03-28T15:04:13Z",
+                                 :resource_version => "150569",
+                                 :labels           => [
+                                   {
+                                     :section => "labels",
+                                     :name    => "department",
+                                     :value   => "Warp-drive",
+                                     :source  => "kubernetes"
+                                   }
+                                 ],
+                                 :tags             => [
+                                   {
+                                     :ns                   => "/managed",
+                                     :category             => "kubernetes:department",
+                                     :category_description => "Kubernetes label 'department'",
+                                     :name                 => "warp_drive",
+                                     :description          => "Warp-drive",
+                                     :read_only            => true,
+                                     :single_value         => true
+                                   }
+                                 ])
```

Comment:
It might be a little nicer to put these hashes into variables and then have something like 
```expect(parser.send(example)).to eq(expected_result)```